api/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path, include
+from . import views
+
+
+urlpatterns = [
+    path('api-auth/', include('rest_framework.urls')),
+    path('currently_playing/', views.currently_playing),
+    path('currently_playing_with_schedule/', views.currently_playing_with_schedule),
+    path('movies/', views.MovieSearchView.as_view()),
+    path('create-hall/', views.create_hall),
```

Comment:
For cases like these, you might want to unify some views based on the resource they manage; for instance, you can implement all CRUD operations under the same class view and expose it via a single path. You can either use DRF's view classes from here https://www.django-rest-framework.org/api-guide/generic-views/#concrete-view-classes, or implement whatever combination of methods you need using the APIView (https://www.django-rest-framework.org/api-guide/views/#class-based-views).

On the long run, with a bigger project the urls file might get too big and it might be nice to keep them clean. This is a big win of class based views over function based views.